src/README.md

```diff
@@ -0,0 +1,173 @@
+# x86-simd-sort
+
+C++ header file library for SIMD based 16-bit, 32-bit and 64-bit data type
+sorting algorithms on x86 processors. We currently have AVX-512 and AVX2
+(32-bit and 64-bit only) based implementation of quicksort, quickselect &
+partialsort and AVX-512 implementations of argsort, argselect and key-value
+sort. The following API's are currently supported:
+
+#### Quicksort
+
+Equivalent to `qsort` in
+[C](https://www.tutorialspoint.com/c_standard_library/c_function_qsort.htm) or
+`std::sort` in [C++](https://en.cppreference.com/w/cpp/algorithm/sort).
+
+```cpp
```

Comment:
I think this should mention the hasnan parameter and its behavior

Reply:
good catch, fixed it.